dd-sdk-android/src/test/kotlin/com/datadog/android/log/internal/logger/LogcatLogHandlerTest.kt

```diff
@@ -84,4 +86,57 @@ internal class LogcatLogHandlerTest {
                     "\$runnable\$1"
             )
     }
+
+    @Test
+    fun `resolves valid stack trace element when wrapped in timber`(forge: Forge) {
+
+        // Given
+        val forgeFileName: Forge.() -> String = {
+            "${this.anAlphabeticalString(Case.ANY)}.${this.aStringMatching("(kt|java)")}"
+        }
+
+        val ignoredElements = forge.aList {
+
+            val className = if (aBool()) {
```

Comment:
class name and method name formats may not cover all the possible naming variants per JLS (where almost everything is allowed), so it covers only a simple naming scheme. let me know if I should widen name generation.

Reply:
I think for me this unit test is enough for now.